Source/CsDebugScript.Engine/UserType.cs

```diff
@@ -71,6 +85,60 @@ public static bool Inherits<T>(this Variable variable)
         {
             return variable.GetRuntimeType().Inherits<T>();
         }
+
+       /// <summary>
+        /// Reinterpret Cast, chanches underlaying code type.
+        /// </summary>
+        /// <typeparam name="T"></typeparam>
```

Comment:
Please fill XML documentation. It is public function and documentation will be generated for it...